test/presenters/return-requirements/start-date.presenter.test.js

```diff
@@ -7,74 +7,89 @@ const Code = require('@hapi/code')
 const { describe, it, beforeEach } = exports.lab = Lab.script()
 const { expect } = Code
 
+// Test helpers
+const DatabaseSupport = require('../../support/database.js')
+const SessionHelper = require('../../support/helpers/session.helper.js')
+
 // Thing under test
 const StartDatePresenter = require('../../../app/presenters/return-requirements/start-date.presenter.js')
 
 describe('Start Date presenter', () => {
-  let session
+  describe('when provided with an unpopulated session', () => {
+    let session
+
+    beforeEach(async () => {
+      await DatabaseSupport.clean()
```

Comment:
If we _do_ need a session record that was persisted to the DB we normally put `await DatabaseSupport.clean()` in a `beforeEach()` at the very top level. That way we know it's always called before any test and saves a bit of duplication.